srcpkgs/python-pgmigrate/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'python-pgmigrate'
+pkgname=python-pgmigrate
+version=1.0.3
+revision=1
+noarch=yes
+wrksrc="pgmigrate-${version}"
+build_style=python-module
+pycompile_module="yandex_pgmigrate"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python-future python-sqlparse python-psycopg2 python-yaml"
+short_desc="Simple tool to evolve PostgreSQL schema easily. (Python2)"
```

Comment:
Remove the dot, xlint didn't complain because of the suffix.
https://github.com/chneukirchen/xtools/blob/master/xlint#L227